lib/drives/pages/drive_detail_page.dart

```diff
@@ -353,6 +371,25 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
     await _drive?.cancel();
     setState(() {});
   }
+
+  int? _getMessageCount(Drive drive) {
```

Comment:
how would that be null? that makes no sense to me... If there are no messages, then it should be zero.

Reply:
yeah this method right now is just a placeholder the real method will be provided when ich finish the chat. I'll leave it for now if that's okay with you

Reply:
sure but the placeholder can still be accurate -> i.e. it cannot return null so the signature is wrong

Reply:
yes that right i changed it